build.sbt

```diff
@@ -470,7 +471,12 @@ lazy val tests = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     ),
     buildInfoPackage := "scala.meta.tests",
     libraryDependencies += "com.lihaoyi" %%% "fansi" % "0.2.5" % "test",
-    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % "test"
+    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % "test",
+    testOptions.in(Test) += Tests.Argument("-l", "org.scalatest.tags.Slow"),
+    inConfig(Fast)(Defaults.testTasks),
```

Comment:
Is `Fast` necessary? Seems like `testsJVM/test` runs the same

Reply:
I included it for consistency. We could remove it if you'd like.

Reply:
Let's remove it to avoid "two ways to do the same thing". Let's also update the "This is a big project with a lot of tests :)" message in build.sbt to reflect the current state of things

> - testsJVM/test      # run fast unit tests
> - testsJVM/slow:test # run slow unit tests

Reply:
We can remove `testQuick` from that too.